docs/greek.rst

```diff
@@ -5,6 +5,42 @@ Greek is an independent branch of the Indo-European family of languages, native
 
 .. note:: For most of the following operations, you must first `import the CLTK Greek linguistic data <http://docs.cltk.org/en/latest/importing_corpora.html>`_ (named ``greek_models_cltk``).
 
+Corpus Readers
```

Comment:
Could this reader evolve into something that takes plain Python str, too? Thus it would begin to look like what we have been calling "the SpaCy-like object". Assuming such an interface is desirable (and to me, it still is), would an inherited NLTK reader be capable of eg:

``` python
processed_latin_text = str_reader("Gloriosissimam ciuitatem Dei siue in hoc temporum cursu", language="latin")
```

I don't remember seeing this in the NLTK docs and it doesn't come up in a cursory search.

cc @free-variation